pixi.toml

```diff
@@ -158,3 +158,11 @@ pre-commit = "*"
 [feature.lint.tasks]
 lint = 'pre-commit run --all-files'
 lint-install = 'pre-commit install'
+
+# =================== TYPE CHECKING ===================
+
+[feature.mypy.dependencies]
+mypy = ">=1.15.0"
+
+[feature.mypy.tasks]
+test-type = "mypy colorcet --strict"
```

Comment:
Cannot `--strict` be instead configured in pyproject.toml?

Reply:
Yes it can. will do.